### PR TITLE
[MODULAR] Adds the suppressor to NT armories.

### PR DESCRIPTION
**Pull request by RatFromTheJungle**
## About The Pull Request
Originally in Scarborough, and made non-contraband for gun cargo, just to be lost to the tides when scarborough became emag only.

## How This Contributes To The Skyrat Roleplay Experience

suppressed peacekeeper shotguns do more damage, and no one can tell me otherwise.

## Changelog

:cl:
add: NT Armories now carries suppressors, for all your indoor shooting needs.
/:cl:

### CODE REVIEW

**Review by Zonespace27 — CHANGES_REQUESTED** [review]
Make the NT suppressor a subtype with a different desc and maybe name, give that to whoever is given one in command already

File: modular_skyrat/modules/projectiles/code/guns/ballistic.dm [hunk 1]
```diff
@@ -0,0 +1,3 @@
+/obj/item/suppressor/nanotrasen
```
Comment (Zonespace27) [inline modular_skyrat/modules/projectiles/code/guns/ballistic.dm]: why is this defined in two different places

File: modular_skyrat/modules/blueshield/code/blueshield.dm [hunk 2]
```diff
@@ -86,7 +86,7 @@
 	new /obj/item/ammo_box/magazine/multi_sprite/cmg(src)
 	new /obj/item/ammo_box/magazine/multi_sprite/cmg/lethal(src)
 	new /obj/item/ammo_box/magazine/multi_sprite/cmg/lethal(src)
-	new /obj/item/suppressor(src) //why the fuck does BS need a suppressed gun is beyond me
+	new /obj/item/suppressor/nanotrasen(src) //why the fuck does BS need a suppressed gun is beyond me
```
Comment (Zonespace27) [inline modular_skyrat/modules/blueshield/code/blueshield.dm:89]: forgot to tick the code files, web edits are bad.

**Review by Zonespace27 — CHANGES_REQUESTED** [review]

**Comment by Gandalf2k15** [comment]
How 👏 many 👏 commits

**Pull request merged**